source/MRViewer/ImGuiHelpers.h

```diff
@@ -186,4 +187,12 @@ MRVIEWER_API PaletteChanges Palette(
     float max = std::numeric_limits<float>::max(),
     const char* format = "%.3f" );
 
+
+/// draw image reversed (corrects its orientation)
+MRVIEWER_API void Image( const MR::ImGuiImage& image, const ImVec2& size, const MR::Color& multColor = MR::Color::white() );
+MRVIEWER_API void Image( const MR::ImGuiImage& image, const ImVec2& size, const ImVec4& multColor );
```

Comment:
better add default value to `const ImVec4&` version instead of `const MR::Color&`